test_scripts/PROJECTION/000_Start_secured_audio_service.lua

```diff
@@ -0,0 +1,63 @@
+---------------------------------------------------------------------------------------------------
```

Comment:
@GetmanetsIrina As far as I understand if PROJECTION is replaced to NAVIGATION all scripts should PASSed. Please confirm.

Reply:
@dboltovskyi , you are right, confirm.